ext/random/tests/03_randomizer/user_unsafe.phpt

```diff
@@ -50,32 +50,280 @@ $randomizer = (new \Random\Randomizer(
 try {
     $randomizer->getInt(\PHP_INT_MIN, \PHP_INT_MAX);
 } catch (\RuntimeException $e) {
-    echo "{$e->getMessage()}\n";
+    var_dump($e);
 }
 
 try {
     $randomizer->getBytes(1);
 } catch (\RuntimeException $e) {
-    echo "{$e->getMessage()}\n";
+    var_dump($e);
 }
```

Comment:
I think I was unclear. Not the exception should be dumped. The value itself should be dumped to ensure there is always output even in case the method does not throw an exception:

```
try {
    var_dump($randomizer->getBytes(1));
} catch (\RuntimeException $e) {
    echo $e->getMessage(), PHP_EOL;
}
```

Reply:
Note: previous to the rebase it was even "var_dump(**bin2hex(**$randomizer->getBytes(1)**)**);" (for ->getBytes(), and ->getInt() was dumped directly)

Reply:
Ah, yes. It needs to be `bin2hex()`, because the byte is not printable otherwise.

Reply:
@zeriyoshi Before you continue to fix the test: Please have a look at this :smiley:

Reply:
Oops! thanks!

Reply:
fixed: 6258125fd9fd9d4f15c28011e13e24be7ea4206e